test/js/css-url-image-value.js

```diff
@@ -1,4 +1,13 @@
 suite('CSSURLImageValue', function() {
+  setup(function() {
+    this.element = document.createElement('div');
```

Comment:
Hmm.. If I use `document.documentElement`, the background of `runner.html` will also change. I can also use the `border-image-source` instead of `background-image`.